modules/drivers/robosense/BUILD

```diff
@@ -0,0 +1,63 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "librobosense_driver_component.so",
+    linkopts = ["-shared"],
```

Comment:
This line should be removed. See #12349 for a reference.
And, what about introducing a  `ROBOSENSE_COPTS` variable to be shared across the whole BUILD file, just like 
'HESAI_COPTS' in the forementioned PR.